### PR TITLE
Realtime notification - Localization

### PR DESCRIPTION
**Pull request by hoanghuee**
## Related Tickets
- [#40159](https://edu-redmine.sun-asterisk.vn/issues/40159)

## WHAT (optional)
Realtime notification - Localization
some Package config files 
## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Review by quanglx-noovado — APPROVED** [review]

**Comment by hoanghuee** [comment]
ready

File: resources/views/vendor/datatables/print.blade.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <style>
+            body {margin: 20px}
+        </style>
+    </head>
+    <body>
+        <table class="table table-bordered table-condensed table-striped">
+            @foreach($data as $row)
+                @if ($loop->first)
+                    <tr>
+                        @foreach($row as $key => $value)
```
Comment (huyenttm-0901) [inline resources/views/vendor/datatables/print.blade.php]: convention

**Comment by hoanghuee** [comment]
ready

File: resources/views/vendor/datatables/print.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        @endif
+        <tr>
+            @foreach ($row as $key => $value)
+            @if (is_string($value) || is_numeric($value))
+            <td>{!! $value !!}</td>
+            @else
+            <td></td>
+            @endif
+            @endforeach
+        </tr>
```
Comment (huyenttm-0901) [inline resources/views/vendor/datatables/print.blade.php]: indent 4 spaces
sao bây giờ cứ mắc lỗi convention thế Huê?

**Comment by hoanghuee** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**